src/components/AddressDetailExplorer.js

```diff
@@ -253,7 +249,11 @@ class AddressDetailExplorer extends React.Component {
             const keys = Object.keys(tokens);
             if (keys.length === 0) {
               // In case the length is 0, we have no transactions for this address
-              this.setState({ loadingTokens: false });
+              this.setState({
+                loadingTokens: false,
+                loadingSummary: false,
+                loadingHistory: false,
+              });
```

Comment:
This is meant to turn all loading flags off, this way the loading icon will not be shown and the error message can be shown.